### ISSUE TITLE
Train is not work? Killed python train.py

### ISSUE THREAD

**Issue by SolicTous** [RaphaelMeudec/tf2-deblurgan-v2#2]
I have tried to train on GoPro but was surprised that the dataset here will be cached.
Now I've just tried on a small GoPro but otherwise, the train will stop with the message:
2020-10-30 17:41:57.484072: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 59 of 200
2020-10-30 17:42:07.441102: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 106 of 200
2020-10-30 17:42:17.419229: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 153 of 200
2020-10-30 17:42:27.465863: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.

It repeats several times.

/home/developer/PycharmProjects/DeBlurGANv2-TF/train.sh: line 1:  7296 Killed                  python train.py

What is that?

What I have to do on the train? Can you provide weights?

I'm interested to do DeBlur on TF2, but convert from DeBlurGANv2 on PyTorch to Pb in TensorFlow does not work(
